devito/finite_differences/tools.py

```diff
@@ -233,7 +233,7 @@ def make_stencil_dimension(expr, _min, _max):
     Create a StencilDimension for `expr` with unique name.
     """
     n = len(expr.find(StencilDimension))
-    return StencilDimension(name='i%d' % n, _min=_min, _max=_max)
+    return StencilDimension('i%d' % n, _min, _max)
```

Comment:
Nitpick: could these just be `min` and `max` now?

Reply:
they are special python keywords so we tend not to as it's not recommended